sdk/python/pulumi_google_native/apigateway/v1/api.py

```diff
@@ -17,12 +17,12 @@ def __init__(__self__, *,
                  location: pulumi.Input[str],
                  project: pulumi.Input[str],
                  display_name: Optional[pulumi.Input[str]] = None,
-                 labels: Optional[pulumi.Input[Mapping[str, pulumi.Input[str]]]] = None,
+                 labels: Optional[pulumi.Input[Mapping[str, str]]] = None,
```

Comment:
This change looks suspicious. I haven't tracked down why. @mikhailshilkov, is this an expected change? If not, it may be a regression from https://github.com/pulumi/pulumi/pull/7059 cc: @pgavlin

Reply:
Looks like there are similar changes to the Node.js SDK.

Reply:
I wouldn't expect this, no. Did it not happen in other providers?

Reply:
This is definitely not expected. What does the schema look like for this property?

Reply:
It's defined as

```
"labels": {
  "type": "object",
  "description": "Optional. Resource labels to represent user-provided metadata. Refer to cloud documentation on labels for more details. https://cloud.google.com/compute/docs/labeling-resources"
},
```

Reply:
Are we tracking this issue somewhere?

Reply:
We are not. I just tried a standalone repro for this locally and got the output I would expect: `Optional[pulumi.Input[Mapping[str, pulumi.Input[str]]]]`

I'll try pulling this branch down and reproing within context.

Reply:
Okay, I can repro this locally. Digging in now.

Reply:
Alright, I have a fix.

Reply:
Fix is part of https://github.com/pulumi/pulumi/pull/7475